README.md

```diff
@@ -1,5 +1,5 @@
 # Ansible VS Code Extension by Red Hat
-This extension adds language support for Ansible to [Visual Studio Code](https://code.visualstudio.com/), via the [ansible-language-server](https://github.com/ansible/ansible-language-server).
+This extension adds language support for Ansible to [Visual Studio Code](https://github.com/ansible/vscode-ansible#:~:text=marketplace.visualstudio.com/items%3Fitemname%3Dredhat.ansible) and [OpenVSX](https://open-vsx.org/extension/redhat/ansible) compatible editors by leveraging [ansible-language-server](https://github.com/ansible/ansible-language-server).
```

Comment:
This is a weird link. Is it intentional?

Reply:
Nope, i think browser copy/paste error. I think i fixed it now.